internal/ingestion/knowledge_compile/consumer.go

```
@@ -895,6 +996,99 @@ func (c *Consumer) processBatch(ctx context.Context, tenant, kb, token string, e
 	return nil
 }
 
+func productContents(products []kccommon.Product) []string {
```

golangci-lint (2.12.2)

[error] func productContents is unused

(unused)